Automattic/newspack-plugin
# Fix new linting errors

### All Submissions:






### Changes proposed in this Pull Request:



This just fixes some linting errors caused by files added between PR #424 was added and merged.

### How to test the changes in this Pull Request:

1. run `npm run lint`
2. command should exit without error
3. also, positive linting result from CI should be reported down there 👇 

### Other information:
